conda/conda-lock.yml

```diff
@@ -9,17 +9,18 @@
 # To update a single package to the latest version compatible with the version constraints in the source:
 #     conda-lock lock --lockfile conda-lock.yml --update PACKAGE
 # To re-solve the entire environment, e.g. after changing a version constraint in the source file:
-#     conda-lock -f /home/leong81/Documents/github/CryoCloudWebsite/conda/environment.yml --lockfile conda-lock.yml
+#     conda-lock -f /home/leong81/Documents/github/CryoCloudWebsite/conda/environment.yml -f /Volumes/GoogleDrive/.shortcut-targets-by-id/0B9H-XP5YRibEY1Brdk1qTzdxUm8/LandsatSST/Scripts/CryoCloudWebsite/conda/environment.yml --lockfile conda-lock.yml
```

Comment:
Not sure how this got here, but might want to remove this

```suggestion
#     conda-lock -f /home/leong81/Documents/github/CryoCloudWebsite/conda/environment.yml --lockfile conda-lock.yml
```

Reply:
My computer made a weird reference to my local folder. My computer was having weird file and permission errors. But it should just be the source of the env.yml file that was used so your file system was added and then mine was.

Reply:
Oops, just saw my username in there :sweat: Should use a generic path instead, but let's do it later...